src/qw/cli.py

```diff
@@ -268,10 +313,14 @@ def freeze(
     service = get_service(conf)
     change_handler = ChangeHandler(service, store)
     diff_elements = change_handler.diff_remote_and_local_items()
+    determiner = LocalChangeNone() if dry_run else LocalChangeInteractive()
```

Comment:
this is nice

Reply:
Thanks, I wanted to leave it open for "accept all" type flags in the future.